modules/ipi-install-configuring-host-dual-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -6,10 +6,16 @@
 [id="configuring-host-dual-network-interfaces-in-the-install-config-yaml-file_{context}"]
 = Optional: Configuring host network interfaces for dual port NIC
 
+Before installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState to support dual port NIC.
+
 :FeatureName: Support for Day 1 operations associated with enabling NIC partitioning for SR-IOV devices
 include::snippets/technology-preview.adoc[leveloffset=+1]
 
-Before installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState to support dual port NIC.
+{VirtProductName} only supports the following bond modes:
+
+* mode=1 active-backup +
```

Comment:
Looks like from the config example that the mode is actually just the `active-backup` part, for example, so should there be some kind of punctuation after the "mode=x" part? 

Example:

* mode=1: active-backup

(I'm not familiar with this subject so maybe not, just a suggestion!)

Reply:
Confirmed with the SME that the pluses are not related to the bond names but instead are backend Asciidoc rendering that might be needed as an `=` and a `:` are used.